src/lib/util/Utils.ts

```diff
@@ -14,16 +14,8 @@ const stringArrayTypeGuard = (object): object is string[] => {
 	return !object.some(entry => typeof entry !== 'string');
 }
 
-const booleanTypeGuard = (object): object is boolean => {
-	return typeof object === 'boolean';
-}
-
-const numberTypeGuard = (object): object is number => {
-	return typeof object === 'number';
-}
-
```

Comment:
These methods are unnecessary, since `@salesforce/ts-types` has equivalents.
The string array typeguard stays because `ts-types`'s `isArray` method doesn't actually enforce the array's contents.